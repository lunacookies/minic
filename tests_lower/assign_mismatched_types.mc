func main {
	n := 5
	np := &n
	set n = np
	set np = n
}
