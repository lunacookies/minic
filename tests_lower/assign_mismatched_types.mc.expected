func main
	var n i64
	var np *i64
	{
		set n = 5
		set np = &(n)
		<missing>
		<missing>
	}
tests_lower:39..41: error: expected “i64” but found “*i64”
tests_lower:52..53: error: expected “*i64” but found “i64”
tests_lower:21..29: warning: unused variable
